client/src/_utils/_routes/AuthenticatedRoute.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { Route } from 'react-router-dom'
+import { LoggedInLayout } from '_shared'
+import { useHistory } from 'react-router-dom'
+import { IsAuthenticated } from '_utils/authenticationHandler'
+
+export default function AuthenticatedRoute({
+  contentComponent: ContentComponent,
+  exact,
+  path,
+  title,
+  // permissions,
+  ...routeProps
+}) {
+  exact = !!exact
+  let history = useHistory()
+  const authenticated = IsAuthenticated
+
+  useEffect(() => {
```

Comment:
I'm curious about this `useEffect` here. Without a dependency array, does it run every time the function is re-rendered? So it's the same as not being in a use Effect?

Reply:
Good point, it's probably not necessary to do it in the useEffect, gonna test that theory.